src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(x) {
+    return function(y) {
+      count++;
+
+      if (count % 2 === 0 && count > 3) {
+        return `Bzzz... Error!`;
+      } else {
+        return x + y;
+      }
```

Comment:
```suggestion
      if (count % 2 === 0 && count > 3) {
        return `Bzzz... Error!`;
      }
      
      return x + y;
      
```

Reply:
avoid using `if else` nesting where possible

Reply:
Thanks for the correction